charmcraft.yaml

```diff
@@ -6,10 +6,10 @@ type: "charm"
 bases:
   - build-on:
     - name: "ubuntu"
-      channel: "20.04"
+      channel: "22.04"
     run-on:
     - name: "ubuntu"
-      channel: "20.04"
```

Comment:
Is this okay now ? (cf: https://github.com/canonical/discourse-k8s-operator/pull/178#pullrequestreview-1829558666)

Reply:
I don't know if it is OK yet, but it is required for the SAML library to work (and building in 22.04 and running in 20.04 does not do the trick). Will ask around.

Reply:
Out of interest can you include some info on what's failing on 20.04? Maybe we can help get it running on that version.

Reply:
Sure thing! The charm fails because of type hints on the library that are not compatible with the Python version shipped with Focal e.g. https://pastebin.canonical.com/p/dSNgsQHsJQ/

Reply:
Ah great, thanks. We own that repository/library so we can potentially propose a change to https://github.com/canonical/saml-integrator-operator to make it focal compatible if we want to go that path.